unit_tests/test_s3_remote_uploader.py

```diff
@@ -0,0 +1,141 @@
+import os
+import pathlib
+import tempfile
+import time
+import subprocess
+import boto3
+import pytest
+import logging
+
+from sdcm.utils.s3_remote_uploader import upload_remote_files_directly_to_s3
+from sdcm.utils.common import S3Storage
+
+# Constants for the test
+TEST_S3_BUCKET = S3Storage.bucket_name
+TEST_S3_KEY_PREFIX = os.environ.get("TEST_S3_KEY_PREFIX", "integration-test/")
+SSH_IMAGE = "linuxserver/openssh-server:latest"
+SSH_USER = "sct-user"  # linuxserver/openssh-server default user
+SSH_PORT = 2222
+
+
+@pytest.fixture(scope="module")
+def ssh_docker_server(tmp_path_factory):
```

Comment:
this should be move to shared location, since more test that uses SSH might need this for testing